JUnit5-README.md

```diff
@@ -0,0 +1,193 @@
+# JUni5 extension
```

Comment:
> JUnit5-README.md

Conceptually I am failing to understand how `JUnit5` made it to set `azure`, `ec2`, `openstack`, `baremetal`, `docker` etc.